src/main/java/com/jc/tm/database/dao/TaskDao.java

```diff
@@ -2,20 +2,31 @@
 
 import com.jc.tm.database.entity.Task;
 
-public interface TaskDao {
+import java.util.List;
 
-    //create or insert task
-    void insetTask(Task task);
+public interface TaskDao extends BaseDao<Task> {
+    @Override
+    default void insert(Task task) {
```

Comment:
do we need defaultt implementaiton of interface?

Reply:
Default was deleted.